src/classes/ACCT_IndividualAccounts_TEST.cls

```diff
@@ -1745,13 +1745,13 @@ private class ACCT_IndividualAccounts_TEST {
                 String FirstName3= 'ConThirdName ' + i;
                 String LastName = 'ConLastName ' + i;
 
-            	System.assertEquals(LastName + ' (' + FirstName1 + ' and ' +
-                					FirstName3 + ') ' + ' Household', returnAccounts[i].Name);
+                System.assertEquals(LastName + ' (' + FirstName1 + ' and ' +
```

Comment:
@kdy we seem to be relying on the order of the results here. Anything else we can try?

Reply:
@lifewithryan , line 1748, FirstName1 is actually a variable so it's not really specifically calling/asking to return the 1st order of the firstname.